docs/Options/GenerateOptions.md

```diff
@@ -1,6 +1,10 @@
 # Generate Options
 Option switches are case-sensitive, arguments are case-insensitive
 
+* `--profile-file=<path>` (or `-p <path>`)
```

Comment:
`-p=<path>` rather than `-p <path>`?
Likewise for -o

Reply:
TLDR: no

The picocli library treats the `=` as optional in both cases.  However, the recommended POSIX standard is that single-letter options do not use `=`, but long options do.  See https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html for an overview, and https://www.gnu.org/software/tar/manual/html_section/tar_21.html for a full real-world example.

Reply:
Interesting, thanks!